guide/russian/css/properties/background-position-property/index.md

```diff
@@ -11,7 +11,7 @@ localeTitle: Исходная позиция
 ```css
 /* setting background-image of HTML doc */ 
  body { 
-  background-image: url('https://i.imgur.com/6Z2VStD.png'); 
+  background-image: url('https://cdn-media-1.freecodecamp.org/imgr/6Z2VStD.png'); 
```

Comment:
Just putting it out here that we need to get on this when we get to the curriculum PR